config/addon-docs.js

```diff
@@ -0,0 +1,9 @@
+/* eslint-env node */
+'use strict';
+
+const AddonDocsConfig = require('ember-cli-addon-docs/lib/config');
```

Comment:
Need

```js
// eslint-disable-next-line node/no-unpublished-require
```
above line 4 to get tests to pass.

Reply:
👍 Just added this -- thanks!

Reply:
I took a closer look at this rule, and disabling it here is just a workaround.

The more complete solution is to create a `files` entry in our `package.json`. This would make it clear that config/addon-docs.js is not something we publish to consumers of the library, which would cause the unpublished rule to not worry about our use of a devDependency here.

Reply:
(Or instead of `files` we could also use .npmignore, which looks like it may be easier for this use case. Basically we just need to prove that config/addon-docs.js does not get published to npm.)

Reply:
Ah, I see there is already an open issue for this on ember-cli-addon-docs: https://github.com/ember-learn/ember-cli-addon-docs/issues/103

Reply:
And a PR https://github.com/ember-learn/ember-cli-addon-docs/pull/105